assets/scripts/books/api.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const getBooks = function () {
+  return $.ajax({
+    url: config.apiUrl + '/books',
+    method: 'GET',
+    headers: {
+      Authorization: 'Token token=' + store.user.token
+    }
```

Comment:
So I think we want to remove this as we don't have any authorization/authentication